app/css/pages/editor.css

```diff
@@ -304,10 +304,11 @@
 
             section.feedback-pane {
                 --header-height: 32px;
+                @apply pb-16 px-24;
```

Comment:
I moved this from lower down. Always best to keep these declarations at the start of a section as it's hard to understand what they're nested in when they're amongst children.